### ISSUE TITLE
[📄DM] Add text to make the history button switch option 'searchable'

### ISSUE THREAD

**Issue by Palloxin** [aniyomiorg/aniyomi#635]
### Describe your suggested feature

So recently i discovered that the history button can be moved again, so i tried to search the related button in the setting typing "history" and the option didn't come out so i searched it manually and i found it.

https://user-images.githubusercontent.com/75091899/176617781-cc8e4892-2046-44bd-bbf0-f6fde774648b.mp4

![IMG_20220630_092516](https://user-images.githubusercontent.com/75091899/176617999-38723d62-eaf6-4860-bda1-84d57e462fcc.jpg)
After i enabled the switcg of the history button that option came out, but onky because i enabled it.
![IMG_20220630_092823](https://user-images.githubusercontent.com/75091899/176618647-d9d4e456-6924-43ee-9f3e-77f29080b115.jpg)
to sum it up, increse the search range of the search bar in the setting like these kind of contexts

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.12.3.6](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Secozzi** [comment]
It's not showing up in search because it's not a setting
